tests/conftest.py

```diff
@@ -3259,3 +3263,14 @@ def flow_cell_encryption_api(
     )
     flow_cell_encryption_api.slurm_api.set_dry_run(dry_run=True)
     return flow_cell_encryption_api
+
+
+def create_process_response(
+    return_code: int = 0, args: str = "", std_out: str = "", std_err: str = ""
```

Comment:
could use a docstring so the use case is clear

Reply:
Added! 👍